parcellab/common/templates/_poddisruptionbudget.tpl

```diff
@@ -11,13 +11,14 @@
 {{- define "common.poddisruptionbudget" -}}
 {{- $podDisruptionBudget := default (dict "enabled" false) .podDisruptionBudget -}}
 {{- $fullname := default (include "common.fullname" .) .name -}}
-{{- if or .Values.podDisruptionBudget.enabled $podDisruptionBudget.enabled }}
+{{- if and $podDisruptionBudget.enabled (or .Values.podDisruptionBudget.enabled $podDisruptionBudget.enabled) }}
```

Comment:
I think this change is not necessary - the previous version covers all cases, or ?